Bio/SeqIO/__init__.py

```diff
@@ -425,6 +427,7 @@
                      "tab": TabIO.TabIterator,
                      "pir": PirIO.PirIterator,
                      "fastq": QualityIO.FastqPhredIterator,
+                     "fastq-4line": QualityIO.FastqFourLineIterator,
```

Comment:
I was thinking ``"fastq": QualityIO.FastqFourLineIterator,`` without adding yet another format name - but that is open to discussion.

Reply:
Sure, that is something that be easily changed once it's been discussed.